Changes.md

```diff
@@ -1,6 +1,14 @@
 1.0.x.x (relative to 1.0.2.0)
 =======
 
+Features
+--------
+
+- Added a non-modal dialog to view the history of a parameter in the Light Editor.
```

Comment:
Start every item with the relevant context for the user - e.g. "Light Editor : Added popup windows to view parameter history". We should mention how you can open the history window in addition to what you can do when it is open.

Reply:
Fixed in 3a3e0a61ec85c003d82b1873be8eef4f7d5ff6f0